packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -479,23 +472,20 @@ export class GarbageCollector implements IGarbageCollector {
 		const fullGC =
 			options.fullGC ??
```

Comment:
I'm not sure we want this to be able to override (disable) autorecovery, should we switch this from `??` to `||`?

Reply:
The GC options except for `runGC` are not used except for the logger. The only reason they exist is for testing. Maybe we should remove it? And rely on feature flags for testing?

Reply:
Since it's only for testing, it doesn't really matter.

Reply:
Sure